samples/hello/docs-hello-history.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
nitpicking again but we might want to rename this to `docs-hello-playback.json` just in case we do add a `history` endpoint in the future.